opm/grid/polyhedralgrid/grid.hh

```diff
@@ -1142,16 +1142,7 @@ namespace Dune
     {
       if( Seed::codimension == 0 )
       {
-        if( cellGeomTypes_.empty() )
-        {
-          assert( geomTypes( Seed::codimension ).size() == 1 );
-          return geomTypes( Seed::codimension )[ 0 ];
-        }
-        else
-        {
-          assert( static_cast<size_t>(seed.index()) < cellGeomTypes_.size() );
-          return cellGeomTypes_[ seed.index() ];
-        }
+        return geomTypes( Seed::codimension )[ 0 ];
```

Comment:
Shouldn't we keep an assert like `assert( geomTypes( Seed::codimension ).size() == 1 );` or `assert(!geomTypes( Seed::codimension ).empty());`

Reply:
I guess so. Will update.